presto-main/src/main/java/io/prestosql/execution/Input.java

```diff
@@ -30,15 +29,15 @@
 @Immutable
 public final class Input
 {
-    private final CatalogName catalogName;
```

Comment:
CatalogSchemaTableName or io.prestosql.metadata.QualifiedObjectName instead of using these three?

Reply:
I think that's overkill. QualifiedObjectName is not json-serializable, and CatalogSchemaTableName has a weird structure that just muddles things (it has catalog + schema-table as a nested object)